src/OpenLoco/src/Scenario.cpp

```diff
@@ -159,22 +160,15 @@ namespace OpenLoco::Scenario
     }
 
     // 0x00475988
-    static void sub_475988()
+    static void resetRoadObjects()
```

Comment:
I think this should be moved into ScenarioConstruction.cpp and renames preformed on the existing `resetConstruction` to make it clear one is for road and one is track.